src/Security/Voter/Committee/EditDeleteCommitteeFeedItemVoter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace AppBundle\Security\Voter\Committee;
+
+use AppBundle\Committee\CommitteeFeedItemPermissions;
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\CommitteeFeedItem;
+use AppBundle\Security\Voter\AbstractAdherentVoter;
+
+class EditDeleteCommitteeFeedItemVoter extends AbstractAdherentVoter
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function supports($attribute, $subject)
+    {
+        return (CommitteeFeedItemPermissions::EDIT === $attribute
+                || CommitteeFeedItemPermissions::DELETE === $attribute)
```

Comment:
Is it useful to have 2 distinct permissions here ? wouldn't "ADMINISTRATE" or "MANAGE" be enough?